src/client/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -103,6 +103,10 @@ public interface IAppConfig
         /// </remarks>
         IEnumerable<string> ClientCapabilities { get; }
 
+        /// <summary>
+        /// Indicates whether the ADAL cache operations will be invoked.
```

Comment:
I don't know that the customer understands "cache operations".
Maybe just write "Indicate whether the ADAL cache will be serializable/deserializable?

Reply:
Maybe rename as `AdalCacheCompatibility` ?

Reply:
Updated.